packages/core/contracts/oracle/interfaces/VotingInterface.sol

```diff
@@ -62,7 +59,23 @@ abstract contract VotingInterface {
      * commitments that can fit in one transaction.
      * @param commits array of structs that encapsulate an `identifier`, `time`, `hash` and optional `encryptedVote`.
      */
-    function batchCommit(Commitment[] calldata commits) external virtual;
+    function batchCommit(Commitment[] memory commits) public virtual;
```

Comment:
this needed to become public to enable calling internally for the override method.